Jropp/jason-ropp-js-iobp
# Onboarding Project: Polymer-Webpack-Plugin

## What It Does

It properly bundles custom components and javascript files.

It was implemented by Bryan Coulter's genius and/or witchcraft in #45 

Closes #38 

## How To Test

Create a user and save, it should appear immediately on the left side of the screen.

Edit that user and save, it should immediately show the updated information.

Edit a user but do not hit save, then try to create and save a new user. There should be a pop-up that keeps you from saving the new user until you save the open edits on a user.

Edit a user but do not hit save, then try to edit another user. There should be a pop-up that keeps you from saving the new user until you save the open edits on a user.

Edit a user then delete that user. You should now be able to freely edit an existing user or save a new user.

## Notes/Caveats


## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies

None
